worker-document-archetype/src/main/resources/archetype-resources/__rootArtifactId__-container/src/main/config/cfg~caf~worker~DocumentWorkerConfiguration.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2015-2017 Hewlett Packard Enterprise Development LP.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+({
+    workerName: "${workerName}",
+    workerVersion: "${version}",
```

Comment:
I have updated this. 
When a project is generated from the archetype, this is the outcome of the config file:
 workerName: "ConfigTestWorker",
 workerVersion: "${project.version}",

'project.version' is not resolved at the same time $(workerName) is.